react/components/ProductImages/utils/thumbnailPositionEnum.js

```diff
@@ -0,0 +1,20 @@
+import { values, map } from 'ramda'
+
+export const thumbnailsPosition = {
+  DISPLAY_LEFT: {
+    name: 'editor.product-details.thumbnailsPosition.left',
+    value: 'left',
+  },
+  DISPLAY_RIGHT: {
+    name: 'editor.product-details.thumbnailsPosition.right',
+    value: 'right',
+  },
+}
+
+export function getThumbnailsPositionNames() {
```

Comment:
Move all the code of this file to
`react/components/ProductImages/Wrapper.js`

Since it's the only module that requires it.